### DISCUSSION TITLE
Conductivity value for lesion

### DISCUSSION THREAD

**Discussion by frankatimm** [simnibs/simnibs#313] [7.Creating head models]
Hello,

I am currently trying to run a simulation with a head model that contains a lesion. In the table containing the standard conductivity values (https://simnibs.github.io/simnibs/build/html/documentation/conductivity.html) they don't talk about the conductivity values for a lesion. I was wondering if there is a standard value that is usually applied for lesions. 
Thank you in advance for your help. 

Best,
Franka

**Comment by oulap** [comment]
I don't think there are any well established values for lesion conductivities. It also depends on what type of lesions you have in mind (MS/stroke). You can have a look at these articles to get started:

https://www.sciencedirect.com/science/article/pii/S2213158217300906

https://www.sciencedirect.com/science/article/pii/S221315822200136X
